### ISSUE TITLE
Router import should be Routes

### ISSUE THREAD

**Issue by cgatian** [BeastCode/VSCode-Angular-TypeScript-Snippets#38]
snippet: `ng-router-featuremodule` and `ng-router-appmodule`

The import from `@angular/router` should be the following:
```
import { RouterModule, Routes } from '@angular/router';
``


Currently the snippet outputs, where `Router` is not used, but the `Routes` type is.
```
import { NgModule } from '@angular/core';
import { RouterModule, Router } from '@angular/router';
import { CommonModule } from '@angular/common';

const routes: Routes = [
  { path: 'path', component: FeatureComponent }
];

@NgModule({
  imports: [CommonModule, RouterModule.forChild(routes)],
  exports: [RouterModule]
})
export class FeatureRoutingModule {}

```

**Comment by BeastCode** [comment]
Ops, I thought I fixed that, see comment in #35.

**Comment by BeastCode** [comment]
Fixed in v4.3.11
